src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue

```diff
@@ -1,19 +1,21 @@
 <template>
-	<div v-if="loanChannelId && selectedChannelLoanIds.length > 0">
+	<div v-if="loanChannelId">
 		<h2 class="tw-mb-2">
-			{{ selectedChannel.name }}
+			{{ name }}
 		</h2>
 		<p class="tw-mb-2">
-			{{ selectedChannel.description }}
+			{{ description }}
 		</p>
-		<kiva-classic-loan-carousel
-			:is-visible="showCarousel"
-			:loan-ids="selectedChannelLoanIds"
-			:selected-channel="selectedChannel"
-			:show-view-more-card="showViewMoreCard"
-			:lend-now-button="lendNowButton"
-			:show-check-back-message="showCheckBackMessage"
-		/>
+		<div>
+			<kiva-classic-loan-carousel
+				:is-visible="true"
```

Comment:
The carousel actually loads much nicer if we just set this to visible. Since it will go instantly into the loading state.